Tox Error: unknown environment 'py35'

### Steps to reproduce
1. Setup the project on a fresh machine
2. Install the python versions required by tox (I used [pyenv](http://sqa.stackexchange.com/a/15257))
3. Run the tests with `make tests`

### Current behavior

```
$ make tests
pyve/bin/tox 
ERROR: unknown environment 'py35'
make: *** [tests] Error 1
```

### Expected behavior

```
$ make tests
pyve/bin/tox 
GLOB sdist-make: /Users/jared/src/pylint/setup.py
py27 create: /Users/jared/src/pylint/.tox/py27
...
```

### pylint --version output

```
$ pylint --version
pylint 2.0.0, 
astroid 1.4.8
Python 2.7.9 (default, May 12 2015, 18:48:38) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.56)]
```

---

### Related issues

https://github.com/spotify/ramlfications/issues/108

### Suggested fix

Bump tox to 1.9.2.

--- a/Makefile
+++ b/Makefile
@@ -23,7 +23,7 @@ $(PIP):
 	virtualenv $(PYVE)
 
 $(TOX): $(PIP)
-	$(PIP) install tox==1.7
+	$(PIP) install tox~=2.0
 
 
 ifdef TOXENV

